lib/Models/Cesium.ts

```diff
@@ -614,6 +632,23 @@ export default class Cesium extends GlobeOrMap {
       allCesium3DTilesets.forEach(tileset => {
         if (!primitives.contains(tileset)) {
           primitives.add(tileset);
+          // Add event listener for loading of new tiles for 3Dtileset datasources. These events are frequently emitted.
+          const startingListener = tileset.tileLoad.addEventListener(() => {
+            this._updateTilesLoadingIndeterminate(true);
+          });
+
+          //Add event listener for when tiles finished loading for current view. Infrequent.
+          const finishedListener = tileset.allTilesLoaded.addEventListener(
+            () => {
+              this._updateTilesLoadingIndeterminate(false);
+            }
+          );
```

Comment:
As discussed, remove these event listeners in `Cesium.destroy` and set indeterminate loading to finished

Reply:
Implemented